cmd-replay.c

```diff
@@ -239,6 +239,13 @@ static void setup_field(struct opts *opts)
 		s++;
 	}
 
+	if (opts->range.start > 0) {
+		if (opts->range.start_elapsed)
+			add_field(field_table[5]);
+		else
+			add_field(field_table[3]);
+	}
```

Comment:
The range can be set "stop" part only.  Also I'd like to add the time field only if user didn't set replay fields explicitly.  IOW, if user gave `-f` option, it should be honored as is IMHO.